goco-transit/src/services/date-service.js

```diff
@@ -5,16 +5,17 @@
  * @param {datetime} datetime to convert
  */
 const getDate = (datetime) => {
-  let date = new Date(datetime.split("T")[0]);
-  return date.getMonth() + "/" + date.getDate() + "/" + date.getFullYear().toString().substr(-2);
+  console.log(datetime)
```

Comment:
want to remove this log dingus? I've noticed some other garbage showing up in the console as well. Remove these when you're done debugging N0ob

Reply:
no u

Reply:
I won't merge until you fix it

Reply:
fixed